features/in_onderzoek.feature

```diff
@@ -20,31 +20,27 @@ Functionaliteit: in onderzoek
 
     @gba
     Abstract Scenario: vragen om <fields> bij <in onderzoek> in onderzoek
-      Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende persoonsgegevens in de registratie
-      | categorie | burgerservicenummer (01.20) | voornamen (02.10) | voorvoegsel (02.30) | geboortedatum (03.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum einde onderzoek (83.30) |
-      | 1         | 555550001                   | Arnitta           |                     | 19231213              | <waarde>                        | 20120920                       |                               |
-      En de persoon met burgerservicenummer 555550001 heeft de volgende oudergegevens in de registratie
-      | Categorie | Voornamen (02.10) | Geslachtsaanduiding (04.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum einde onderzoek (83.30) |
-      | 2         | Noa               | V                           |                                 |                                |                               |
-      | 3         | Jan               | M                           |                                 |                                |                               |
-      En de persoon met burgerservicenummer 555550001 heeft de volgende partnergegevens in de registratie
-      | Categorie | Burgerservicenummer (01.20) | Voornamen | Geslachtsaanduiding (04.10) | Datum aangaan (06.10) | Datum ontbinding (07.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum einde onderzoek (83.30) |
-      | 5         | 555550002                   | Karel     | M                           | 20091102              |                          |                                 |                                |                               |
+      Gegeven de persoon met burgerservicenummer '555550001' heeft de volgende gegevens
```

Comment:
Deze stap wordt niet ondersteund. Hiervoor kan `Gegeven een persoon heeft de volgende 'persoon' gegevens` stap worden gebruikt. De bsn kan dan als kolom worden opgenomen in de tabel